package.json

```diff
@@ -52,20 +54,29 @@
     "@babel/preset-env": "^7.22.9",
```

Comment:
These dependencies can be dropped.

Reply:
Are you certain? The options we pass to prettier include `{ parser: 'babel' }`.
If prettier ships with its own babel parser stuff I guess that's fine.

I pushed the PR with the babel dependencies removed, and the tests are passing, so hopefully everything is peachy.